[Java Client - Retrofit 2] Petstore sample does not build when 'target = android'

Here is the error : 

```
C:\development\projects\android-swagger-petstore-sdk-retrofit2\src\main\java\io\swagger\client\StringUtil.java:3: error: package javax.annotation does not exist
@javax.annotation.Generated(value = "class io.swagger.codegen.languages.JavaClientCodegen", date = "2016-03-14T22:17:54.660+08:00")
                 ^
Note: C:\development\projects\android-swagger-petstore-sdk-retrofit2\src\main\java\io\swagger\client\ApiClient.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
:compileDebugJava FAILED
```

I think there is a missing dependency (which, by the way, is present in Retrofit 1 sample) : 

```
dependencies {
            provided 'javax.annotation:jsr250-api:1.0'
}
```

--- a/modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/build.gradle.mustache
+++ b/modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/build.gradle.mustache
@@ -46,6 +46,10 @@ if(hasProperty('target') && target == 'android') {
 	            }
 	        }
 	    }
+
+      dependencies {
+          provided 'javax.annotation:jsr250-api:1.0'
+      }
 	}
 
 	afterEvaluate {

--- a/samples/client/petstore/java/retrofit2/build.gradle
+++ b/samples/client/petstore/java/retrofit2/build.gradle
@@ -46,6 +46,10 @@ if(hasProperty('target') && target == 'android') {
 	            }
 	        }
 	    }
+
+      dependencies {
+          provided 'javax.annotation:jsr250-api:1.0'
+      }
 	}
 
 	afterEvaluate {

